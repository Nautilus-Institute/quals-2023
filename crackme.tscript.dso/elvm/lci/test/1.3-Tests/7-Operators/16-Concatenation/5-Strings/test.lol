HAI 1.3
	VISIBLE SMOOSH "a" AN "" AN "b" AN "WIN" AN "c" AN "1" AN "d" AN "3.456" AN "e" MKAY
KTHXBYE
