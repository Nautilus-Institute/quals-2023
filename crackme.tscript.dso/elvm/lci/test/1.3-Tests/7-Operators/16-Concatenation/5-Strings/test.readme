This test makes sure the concatenation operator works correctly on string
arguments.
