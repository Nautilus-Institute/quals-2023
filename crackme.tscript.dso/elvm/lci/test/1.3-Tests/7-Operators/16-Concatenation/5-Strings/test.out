abWINc1d3.456e
